virtool/samples/db.py

```diff
@@ -74,12 +74,13 @@
 }
 
 
-async def attach_artifacts_and_reads(pg: AsyncEngine, document: dict) -> dict:
+async def attach_artifacts_and_reads(pg: AsyncEngine, document: dict, no_download_urls: bool = False) -> dict:
```

Comment:
Why is `no_download_urls` necessary?

They should be included in all responses, but not stored in the database. Maybe I'm missing something.